build-system/server.js

```diff
@@ -556,6 +556,51 @@ app.use('/bind/form/get', function(req, res, next) {
   });
 });
 
+// Simulated Cloudflare signed Ad server
+
+cloudflareDir = '/extensions/amp-ad-network-cloudflare-impl/0.1/data';
+
+/**
+ * Handle CORS headers
+ */
+app.use(cloudflareDir, function fakeCors(req, res, next) {
+    res.setHeader("Access-Control-Allow-Methods", "GET,OPTIONS");
+    res.setHeader("Access-Control-Allow-Origin", req.get("Origin") || "*");
+    res.setHeader("AMP-Access-Control-Allow-Source-Origin", req.params["__amp_source_origin"] || "");
+    res.setHeader("Access-Control-Expose-Headers", "AMP-Access-Control-Allow-Source-Origin,X-AmpAdSignature");
+    next();
+});
+
+/**
+ * Handle CORS preflight
+ */
+app.options(cloudflareDir + '/*', function(req, res) {
+    res.status(204);
+    res.end();
+});
+
+/**
+ * Handle fake data
+ */
+app.get(cloudflareDir + '/*', function(req, res) {
```

Comment:
This one is different, since it has different header handling than the fake one.